client/java/src/main/java/io/openlineage/client/circuitBreaker/TaskQueueCircuitBreakerConfig.java

```diff
@@ -22,8 +22,8 @@
 public class TaskQueueCircuitBreakerConfig
     implements CircuitBreakerConfig, MergeConfig<TaskQueueCircuitBreakerConfig> {
   private static final Integer DEFAULT_THREAD_COUNT = 2;
-  private static final Integer DEFAULT_QUEUE_SIZE = 2;
-  private static final Long DEFAULT_TIMEOUT = 1L;
+  private static final Integer DEFAULT_QUEUE_SIZE = 10;
+  private static final Long DEFAULT_TIMEOUT = 90L;
```

Comment:
Could you please share motivation behind this change?

Reply:
All other circuit breakers have 90 sec timeout. That's why I wanted to be consistent.